cubequery/api_server.py

```diff
@@ -7,7 +7,7 @@
 from flask import Flask, request, abort, render_template, jsonify, send_file
 from flask_caching import Cache
 from flask_cors import CORS
-from itsdangerous import TimedJSONWebSignatureSerializer as Serializer, SignatureExpired, BadSignature
+from itsdangerous import TimedJSONWebSignatureSerializer as Serializer

```

Comment:
What still ends up using the TimedJSONWebSignatureSerializer? would have expected all of itsdangerous to be able to vanish